media-sound/musescore/musescore-3.2.3.recipe

```diff
@@ -65,17 +65,17 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make release PREFIX=$prefix UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
+	make release PREFIX=$appsDir UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
 		BUILD_JACK=OFF BUILD_PORTAUDIO=OFF BUILD_WEBENGINE=OFF \
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF
 }
 
 INSTALL()
 {
-	make install PREFIX=$prefix UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
+	make install PREFIX=$appsDir UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
 		BUILD_JACK=OFF BUILD_PORTAUDIO=OFF BUILD_WEBENGINE=OFF \
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF
-	rm -rf $prefix/share/icons $prefix/share/applications/mscore.desktop
-	mv $prefix/share $prefix/data
+	rm -rf $appsDir/share/icons $appsDir/share/applications/mscore.desktop
+	mv $appsDir/share $appsDir/data
 	addAppDeskbarSymlink $binDir/musescore Musescore
```

Comment:
This link is now broken. Why move musescore to appsDir? It's a ported application so it uses the standard filesystem organization (with bin/ data/ etc) and I see no point in moving it around, except adding more maintenance effort?

Reply:
The modifications are for issue #3217.

Reply:
If we really want to fix this, we could move just the executable from bin/ to apps/, but leave the other things where they are.

In any case, please fix the addAppDeskbarSymLink to point to the executable, and make sure it's possible to start the application from DeskBar.

Reply:
Default install uses /usr/local, that is overridden by our PREFIX, but "share" is still being used for the data, either we use PREFIX=$appsDir/MuseScore or try to fix the "share" in the build/install part